invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/MobileUploadsItem.js

```diff
@@ -0,0 +1,145 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import _truncate from "lodash/truncate";
+import React from "react";
+import { Button, Icon, Item, Label } from "semantic-ui-react";
+import { SearchItemCreators } from "../../utils";
+
+export const MobileUploadsItem = ({ result, index, editRecord, statuses }) => {
+  const access_status_id = _get(
+    result,
```

Comment:
Could maybe all these shared variables be collected in the parent and passed down to the computer/mobile component, so that we don't need to define them twice?

Reply:
I agree with you, but I got some backlash in another similar PR about having components with a huge number of parameters passed. I would say we need to establish clearly what should and should not get duplicated @kpsherva.